app/helpers/admin/users_helper.rb

```diff
@@ -0,0 +1,21 @@
+module Admin::UsersHelper
+  def get_gender(gender)
+    gender == 'M' ? 'Male' : 'Female'
+  end
+  
+  def get_enable_or_disable_link(user)
+    if user.enabled?
+      link_to 'Disable', enable_or_disable_user_admin_user_path(user), data: { toggle: "tooltip" }, title: 'Disable User', remote: true
```

Comment:
something like
```
def status_update_link(status)
    link_to status.humanize, enable_or_disable_user_admin_user_path(user), data: { toggle: "tooltip" }, title: "#{status.humanize} User", remote: true
end
```

Reply:
route name should be changed as per Ammad's comments.

Reply:
Done Sir.